Extensions/PathfindingBehavior/PathfindingBehavior.cpp

```diff
@@ -57,6 +59,11 @@ std::map<gd::String, gd::PropertyDescriptor> PathfindingBehavior::GetProperties(
       gd::String::From(behaviorContent.GetDoubleAttribute("gridOffsetY", 0)));
   properties[_("Extra border size")].SetGroup(_("Collision")).SetValue(
       gd::String::From(behaviorContent.GetDoubleAttribute("extraBorder")));
+  properties[_("Smoothing max cell gap")]
+      .SetValue(gd::String::From(
+          behaviorContent.GetDoubleAttribute("smoothingMaxCellGap")))
+      .SetGroup(_("Path smoothing"))
+      .SetDescription(_("It's recommended to leave a max gap of 1 cell."));
```

Comment:
Is it worth adding a bit more info? Like what should the use expect by increasing/decreasing this value

Reply:
I added a note about what `0` does. Anything other than `0` or `1` is a bit useless. Values less than 1 will only smooth some parts of the path and value over 1 may remove important point. I didn't use a boolean because it might be useful in very rare cases.

Reply:
OK nice!